src/bson_mapper/bson_archiver.hpp

```diff
@@ -45,6 +45,7 @@
 
 #include <chrono>
 #include <memory>
+#include <sstream>
```

Comment:
Removing `<sstream>` gives me `error: implicit instantiation of undefined template 'std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >'`